### ISSUE TITLE
error when rbdl.CalcCenterOfMass()

### ISSUE THREAD

**Issue by zxiaoyusq** [ORB-HD/rbdl-orb#69]
the following code:
`rbdl.CalcCenterOfMass(model, q, qdot, qddot, CoM)`
Gives the following error:
```
File "/home/lmd/dev/physcap_demo_release-master/Utils/core_utils.py", line 86, in get_projected_CoM
    rbdl.CalcCenterOfMass(model, q, qdot, qddot, CoM)
  File "rbdl.pyx", line 3150, in rbdl.CalcCenterOfMass
    if qddot and com_acceleration is not None:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

```
and the value:
![www](https://user-images.githubusercontent.com/29696398/126425160-7e06aa3e-53dd-4213-90aa-c497b6e806a6.jpg)

Could you point where I did wrong? Thank you for your help.

**Comment by ju6ge** [comment]
Hey can you post the code snipped otherwise it is hard to tell what exactly is causing the issue.

**Comment by antoinedelplace** [comment]
Same as #71.
Should be fixed by #72

**Comment by ju6ge** [comment]
closed with #72
